.github/workflows/build.yml

```diff
@@ -176,6 +176,7 @@ jobs:
             -
                 name: Validate composer.json
                 run: composer validate --ansi --strict
+                if: matrix.sylius != '1.12.x-dev'
```

Comment:
We can change 
`sylius: ["~1.11.2", "1.12.x-dev"]`
to
`sylius: ["~1.11.2", "~1.12.x-dev"]`

Then we can drop this `if` statement and we don't have to remember about this later.